#~ pandora:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Position Controllers ---------------------------------------
  laser_roll_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: laser_roll_joint
    pid: {p: 10.0, i: 0.01, d: 10.0}
  laser_pitch_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: laser_pitch_joint
    pid: {p: 10.0, i: 0.01, d: 10.0}
  kinect_pitch_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: kinect_pitch_joint
    pid: {p: 5.0, i: 0.01, d: 0.5}
  kinect_yaw_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: kinect_yaw_joint
    pid: {p: 5.0, i: 0.01, d: 0.5}
